src/utils/url.ts

```diff
@@ -14,8 +14,19 @@ const protocolAndDomainRE = /^(?:\w+:)?\/\/(\S+)$/;
 const localhostDomainRE = /^localhost[:?\d]*(?:[^:?\d]\S*)?$/;
 const nonLocalhostDomainRE = /^[^\s.]+\.\S{2,}$/;
 
+/**
```

Comment:
please remove leftover, and also please retarget the pr to `next-version` branch 🙏

Reply:
ok @ivpavici  i will do so

Reply:
@ivpavici please what do mean by leftover

Reply:
done